IPython/core/completer.py

```diff
@@ -765,9 +775,59 @@ def magic_matches(self, text):
             comp += [ pre+m for m in line_magics if m.startswith(bare_text)]
         return comp
 
+    def python_jedi_matches(self, text):
+        """Match attributes or global Python names using Jedi."""
+        namespaces = []
+        if self.namespace is None:
+            import __main__
+            namespaces.append(__main__.__dict__)
+        else:
+            namespaces.append(self.namespace)
+        if self.global_namespace is not None:
+            namespaces.append(self.global_namespace)
+
+        # From Jedi docs
+        sys.path.insert(0, os.getcwd())
```

Comment:
I'm not sure why that Jedi code does it, but IPython already adds the CWD to `sys.path`, so I'm pretty sure we don't need to do it here.